### ISSUE TITLE
`wtx submit init` not finding `publish-browser-extension`

### ISSUE THREAD

**Issue by leos** [wxt-dev/wxt#479]
### Describe the bug
I'm trying to invoke `wtx submit init` but it's not working - it seems to be looking for `publish-browser-extension` in the wrong place?

I started with the React template if that's useful. I can try to create a repro if this is not something common (my first time with wtx and chrome extensions/typescript/etc.). Let me know if I can provide any other info.

```
~/repos/DVR2/MyProject ❯❯❯ pnpm wxt submit init
node:internal/modules/cjs/loader:1152
  throw err;
  ^

Error: Cannot find module '/Users/xxx/repos/DVR2/publish-browser-extension@2.1.2/node_modules/publish-browser-extension/bin/publish-extension.cjs'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1149:15)
    at Module._load (node:internal/modules/cjs/loader:990:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v21.6.2
```

### Environment

```
  System:
    OS: macOS 14.3.1
    CPU: (12) arm64 Apple M2 Pro
    Memory: 267.41 MB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 21.6.2 - /opt/homebrew/bin/node
    npm: 10.2.4 - /opt/homebrew/bin/npm
    pnpm: 8.15.3 - /opt/homebrew/bin/pnpm
  Browsers:
    Chrome: 122.0.6261.57
    Safari: 17.3.1
  npmPackages:
    wxt: ^0.16.0 => 0.16.9
```

**Comment by aklinker1** [comment]
Yeah, I can reproduce this, but I can't figure out why it's failing. Something to do with the binary PNPM is creating, it's pointing at the wrong directory to lookup the JS file... My current setup is just to add `node_modules/wxt/node_modules/.bin` (where `publish-extension` is located) to the `PATH` environment variable, but that's causing the messedup path, so I need to think of something else.

Also, this also breaks `wxt build --analyze`, which is also using a subdependencies binary. 

I think I'll lookup the executable path and execute it directly rather than manipulating the `PATH` environment variable.

**Comment by aklinker1** [comment]
Hmm, OK, so same error if I run the binaries directly without modifying the path... I think I'll have to go with the route of asking you to install the package if it can't be found, similar to vitest when you run the coverage command for the first time asking to install the coverage module.

**Comment by aklinker1** [comment]
I wonder what broke.... This used to work, but now it doesn't? Maybe PNPM has updated... Hmm. Either way, asking to install will work, and is the most future-proof I think.

**Comment by aklinker1** [comment]
Hmm, yeah, still can't find a way to figure this out. For now, a workaround:

```ts
pnpm i publish-browser-extension
pnpm publish-extension init
```

`wxt submit` is just an alias for `publish-extension`, so you can run that CLI tool by hand.

**Comment by aklinker1** [comment]
Fixed in [`v0.17.2`](https://github.com/wxt-dev/wxt/releases/tag/v0.17.2)

**Comment by leos** [comment]
Works great, thank you!
